packages/postgres-database/src/simcore_postgres_database/utils_folders.py

```diff
@@ -99,8 +108,15 @@ class CouldNotCreateFolderError(BaseCreateFolderError):
     msg_template = "Could not create folder='{folder}' and parent='{parent}'"
 
 
-class GroupIdDoesNotExistError(BaseCreateFolderError):
-    msg_template = "Provided group id '{gid}' does not exist "
+class GroupIDsDoNotExistError(BaseCreateFolderError):
+    msg_template = "Any of the folloing gids='{gids}' do not exist"
```

Comment:
this error does not seem very useful. so one of the groups in there does not exist? can't we have only the ones that are a problem?

Reply:
Also "following"